asserts/database.go

```diff
@@ -81,31 +113,25 @@ func OpenDatabase(cfg *DatabaseConfig) (*Database, error) {
 	if info.Mode().Perm()&0002 != 0 {
 		return nil, fmt.Errorf("assert database root unexpectedly world-writable: %v", cfg.Path)
 	}
+
+	if be == nil {
+		be = newFilesystemBackstore(cfg.Path)
+	}
+	err = be.Init(buildAssertion)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
The above is all changing given comments. Happy to see it paced as you prefer, though.